pep8.py

```diff
@@ -1890,7 +1898,14 @@ def excluded(self, filename, parent=None):
 
         Check if 'options.exclude' contains a pattern that matches filename.
         """
-        if not self.options.exclude:
+        result = self._excluded(filename, parent=parent)
+        if result and self.options.verbose:
+            print('ignoring excluded file: %s' % filename)
+        return result
+
+    def _excluded(self, filename, parent=None):
```

Comment:
Since this method is the workhorse of `.exclude()` itself, I didn't feel the need at the time since it would just be repeating that method's docstring.

What do you suggest as docstring? Should we change the docstring of `.exclude()` to explain it's just a logging wrapper around `._exclude()` and move that docstring here?